api/src/routers/last-meal.js

```diff
@@ -0,0 +1,16 @@
+import express from "express";
+import knex from "../database_client.js";
+
+const lastMealroute = express.Router();
+
+lastMealroute.get("/", async (req, res, next) => {
+    console.log("Received request for /api/last-meal");
+    try{
+        const last_meals = await knex("meal").where("id",">",0).orderBy("id","desc").first();
```

Comment:
ID is incremental starting from 1so this query is not required
```suggestion
        const last_meals = await knex("meal").orderBy("id","desc").first();
```